docs/feature_backlight.md

```diff
@@ -93,18 +93,18 @@ BACKLIGHT_DRIVER = pwm
 
 On AVR boards, QMK automatically decides which driver to use according to the following table:
 
-|Backlight Pin|AT90USB64/128|ATmega16/32U4|ATmega16/32U2|ATmega32A|ATmega328/P|
-|-------------|-------------|-------------|-------------|---------|-----------|
-|`B1`         |             |             |             |         |Timer 1    |
-|`B2`         |             |             |             |         |Timer 1    |
-|`B5`         |Timer 1      |Timer 1      |             |         |           |
-|`B6`         |Timer 1      |Timer 1      |             |         |           |
-|`B7`         |Timer 1      |Timer 1      |Timer 1      |         |           |
-|`C4`         |Timer 3      |             |             |         |           |
-|`C5`         |Timer 3      |             |Timer 1      |         |           |
-|`C6`         |Timer 3      |Timer 3      |Timer 1      |         |           |
-|`D4`         |             |             |             |Timer 1  |           |
-|`D5`         |             |             |             |Timer 1  |           |
+|Backlight Pin|AT90USB64/128|AT90USB162|ATmega16/32U4|ATmega16/32U2|ATmega32A|ATmega328/P|
+|-------------|-------------|----------|-------------|-------------|---------|-----------|
+|`B1`         |             |          |             |             |         |Timer 1    |
+|`B2`         |             |          |             |             |         |Timer 1    |
+|`B5`         |Timer 1      |          |Timer 1      |             |         |           |
+|`B6`         |Timer 1      |          |Timer 1      |             |         |           |
+|`B7`         |Timer 1      |Timer 1   |Timer 1      |Timer 1      |         |           |
+|`C4`         |Timer 3      |          |             |             |         |           |
+|`C5`         |Timer 3      |Timer 1   |             |Timer 1      |         |           |
+|`C6`         |Timer 3      |Timer 1   |Timer 3      |Timer 1      |         |           |
+|`D4`         |             |          |             |             |Timer 1  |           |
+|`D5`         |             |          |             |             |Timer 1  |           |
```

Comment:
So we have something like this below that looks good using `qmk docs` but a little harder to understand just as text.
```suggestion
|Backlight Pin|AT90USB64/128|ATmega16/32U4|AT90USB162<br>ATmega16/32U2|ATmega32A|ATmega328/P|
|-------------|-------------|-------------|-------------|---------|-----------|
|`B1`         |             |             |             |         |Timer 1    |
|`B2`         |             |             |             |         |Timer 1    |
|`B5`         |Timer 1      |Timer 1      |             |         |           |
|`B6`         |Timer 1      |Timer 1      |             |         |           |
|`B7`         |Timer 1      |Timer 1      |Timer 1      |         |           |
|`C4`         |Timer 3      |             |             |         |           |
|`C5`         |Timer 3      |             |Timer 1      |         |           |
|`C6`         |Timer 3      |Timer 3      |Timer 1      |         |           |
|`D4`         |             |             |             |Timer 1  |           |
|`D5`         |             |             |             |Timer 1  |           |
```


Or something like below but I think shortening it to 162 probably isn't clear enough and adding the full name makes the column extra wide.
```suggestion
|Backlight Pin|AT90USB64/128|ATmega16/32U4|ATmega16/32U2/162|ATmega32A|ATmega328/P|
|-------------|-------------|-------------|-----------------|---------|-----------|
|`B1`         |             |             |                 |         |Timer 1    |
|`B2`         |             |             |                 |         |Timer 1    |
|`B5`         |Timer 1      |Timer 1      |                 |         |           |
|`B6`         |Timer 1      |Timer 1      |                 |         |           |
|`B7`         |Timer 1      |Timer 1      |Timer 1          |         |           |
|`C4`         |Timer 3      |             |                 |         |           |
|`C5`         |Timer 3      |             |Timer 1          |         |           |
|`C6`         |Timer 3      |Timer 3      |Timer 1          |         |           |
|`D4`         |             |             |                 |Timer 1  |           |
|`D5`         |             |             |                 |Timer 1  |           |
```

There's probably another option but my markdown knowledge is non-existent.